tests/runtest.py

```diff
@@ -735,13 +735,13 @@ def run_tests(host_os,
     # Setup the dotnetcli location
     dotnetcli_location = os.path.join(coreclr_repo_location, "Tools", "dotnetcli", "dotnet%s" % (".exe" if host_os == "Windows_NT" else ""))
 
-    # Default timeout for unix is 15 minutes
-    os.environ["__TestTimeout"] = str(15*60*1000) # 900,000 ms
```

Comment:
Why are we changing the default timeout for stress testing?

Reply:
This is what I was asking about above. The GCStress tests are currently running using the default timeout

Reply:
Hm that is interesting. Looks like https://github.com/dotnet/coreclr/blob/master/tests/runtest.py#L539 may be the issue.

Reply:
It should most likely be `if "COMPlus_GCStress" in complus_vars:`

Reply:
Hm...

Reply:
I wonder if this is because we have both `gc_stress` and `gc_stress_c`?

Reply:
This code path only is executed if there is a test env. I have a feeling we have a test env created that is not the format the hacky parser is able to read.

Reply:
Hmmm, no; looks like we always set `gc_stress` when `gc_stress_c` is set.